### DISCUSSION TITLE
Updating title

### DISCUSSION THREAD

**Discussion by lcmen** [unpoly/unpoly#364] [Q&A] [answered]
Hello!
Long time Unpoly fan here, thanks for making this awesome library!

I have a quick question about title updates (from response's `<head>` tag or via `up.title=` with Rails): do I need to configure anything extra if I want to change the title from the response generated by the link with `up-follow` and `up-target`?

I enabled logs (via `up.log.enable()`) on the `console` but I don't see any event related to title.

**Comment by triskweline** [comment, marked as answer]
Hi @lcmen!

By default links and forms will update history (window title and address bar) whenever you're rendering a [main target](https://unpoly.com/main), which is usually your layout's main content area. When other targets are rendered Unpoly will assume it's a minor fragment that should not update history.

You can configure which targets cause history updates in [`up.fragment.config.autoHistoryTargets`](https://unpoly.com/up.fragment.config#config.autoHistoryTargets).

To always update history regardless of target you can set `up.fragment.config.navigateOptions.history = true` (default is `'auto'`).

**Reply by lcmen** [reply]
Thank you @triskweline ! I wasn't aware of the connection between history target and title.

Out of curiosity: is it possible to use `config.navigateOptions.history` to update title without updating URL?
